### PR TITLE
Leaderboard Modal, Leaderboard Enhancements

### PR DESCRIPTION
**Pull request by keithcharlesc**
- Added modal for viewing badge
- Removed view more 
- Added more users for the main list
- Changed name of current user to Steven Lim
- Added more badges collections

### CODE REVIEW

File: components/leaderboard/BadgesCollection.tsx [hunk 1]
```diff
@@ -1,15 +1,68 @@
...
+                  >
+                    <span className='bg-transparent text-black opacity-5 h-6 w-6 text-2xl block outline-none focus:outline-none'>×</span>
+                  </button>
+                </div>
+                {/*body*/}
+                <br />
+                <Image alt='me' width='100' height='100' src={`/badges/carbon_10.svg`} />
+                <div className='relative p-6 pt-0 flex-auto'>
+                  <p className='my-4 text-slate-500 text-lg leading-relaxed'>
+                    Awesome job! You&apos;ve managed to offset <strong>10kg worth of CO2</strong> and played a huge role in reducing the
```
Comment (remuskwan) [inline components/leaderboard/BadgesCollection.tsx:33]: Change 2 in CO2 to superscript haha can try using `<sup>` tag

File: components/leaderboard/BadgesCollection.tsx [hunk 2]
```diff
@@ -1,15 +1,68 @@
-import React from 'react';
+import React, { Fragment, useState } from 'react';
 import Image from 'next/image';
+import { Dialog, Transition } from '@headlessui/react';
+import { CheckIcon } from '@heroicons/react/24/outline';
 
 export default function BadgesCollection(props) {
+  const [showModal, setShowModal] = React.useState(false);
```
Comment (remuskwan) [inline components/leaderboard/BadgesCollection.tsx:7]: `React.` is redundant here since u imported `useState` alr

**Review by remuskwan — CHANGES_REQUESTED** [review]
small changes!

File: components/leaderboard/BadgesCollection.tsx (hunk 2 above)
Comment (keithcharlesc) [reply components/leaderboard/BadgesCollection.tsx]: Done!

**Review by remuskwan — APPROVED** [review]
lgtm

**Pull request merged**